src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -121,21 +121,22 @@ public void deleteMessageWithID(String messageID) {
    * @return a list of messages posted by user, or empty list if no messages posted to country page.
    *     List is sorted by time descending.
    */
-  public List<Message> getMessagesByCategory(String countryCode, String category){
+  public List<Message> getMessagesByCategory(String countryCode, String category) {
 
-	  Query.Filter countryFilter = new Query.FilterPredicate("country", FilterOperator.EQUAL, countryCode);
-	  Query.Filter categoryFilter = new Query.FilterPredicate("category", FilterOperator.EQUAL, category);
+    Query.Filter countryFilter =
+        new Query.FilterPredicate("country", FilterOperator.EQUAL, countryCode);
+    Query.Filter categoryFilter =
+        new Query.FilterPredicate("category", FilterOperator.EQUAL, category);
 
-	  Query.Filter combinedFilter = Query.CompositeFilterOperator.and(countryFilter, categoryFilter);
+    Query.Filter combinedFilter = Query.CompositeFilterOperator.and(countryFilter, categoryFilter);
 
-	  Query query =
-	         new Query("Message")
-	              .setFilter(combinedFilter)
-	              .addSort("timestamp", SortDirection.DESCENDING);
-	      PreparedQuery results = datastore.prepare(query);
-	      Entity userEntity = results.asSingleEntity();
```

Comment:
Just wondering - why was the Entity removed?

Reply:
@gjchong25 it seemed to be causing an error in loading the category page if it had more than one message, I think. Also, because I don't think we used it anywhere else, so I think it was extraneous code. But if we have any problems, I can definitely add it back in